mlflow/sklearn.py

```diff
@@ -171,7 +171,7 @@ def save_model(sk_model, path, conda_env=None, mlflow_model=None,
         yaml.safe_dump(conda_env, stream=f, default_flow_style=False)
 
     pyfunc.add_to_model(mlflow_model, loader_module="mlflow.sklearn",
-                        model_path=model_data_subpath, env=conda_env_subpath)
+                        data=model_data_subpath, env=conda_env_subpath)
```

Comment:
model data should be stored in `data` otherwise `pyfunc.load_model` would try to read model data from `local_path` and fail.

https://github.com/mlflow/mlflow/blob/master/mlflow/pyfunc/__init__.py#L472-L473